README.md

```diff
@@ -111,8 +111,18 @@ A logFile listing all the tickets created can be found where the tool has been r
 ## Jira.yaml
 
 Example of config file structure. 
-If your jira project has custom mandatory field configured, they will need to be added to the config file
-Please make sure you give both key and value expected by jira under the customMandatoryField key of the config file
+If your jira project has specific mandatory field or custom fields configured, they will need to be added to the config file.
+Mandatory fields: 
+  - Make sure to give both key and value expected by jira under the customMandatoryField key of the config file.
+Custom fields:
+  - At the moment we are supporting 3 types of custom fields: labels, MultiGroupPicker and MultiGroupPicker.
```

Comment:
Can you highlight the names like so:`labels`, `MultiGroupPicker` and `MultiGroupSelect`.